packages/shared/tsconfig.json

```diff
@@ -4,7 +4,8 @@
     "esModuleInterop": true,
     "forceConsistentCasingInFileNames": true,
     "isolatedModules": true,
-    "moduleResolution": "node",
+    "moduleResolution": "Bundler",
```

Comment:
We don't use TS to transpile anything: https://www.totaltypescript.com/tsconfig-cheat-sheet#not-transpiling-with-typescript